README.md

```diff
@@ -1543,6 +1545,47 @@ FLAG DESCRIPTIONS
     scripts), you can use this flag to easily get the IDs of created resources for further processing.
 ```
 
+## `mw app upload [INSTALLATION-ID]`
+
+Upload the filesystem of an app to a project
+
+```
+USAGE
+  $ mw app upload [INSTALLATION-ID] --source <value> [-q] [--ssh-user <value>] [--dry-run] [--delete]
+
+ARGUMENTS
+  INSTALLATION-ID  ID or short ID of an app installation; this argument is optional if a default app installation is set
+                   in the context
+
+FLAGS
+  -q, --quiet             suppress process output and only display a machine-readable summary.
+      --delete            delete remote files that are not present locally
+      --dry-run           do not actually upload the app installation
+      --source=<value>    (required) source directory from which to upload the app installation
```

Comment:
Would it be useful to set a default to the current directory if omitted?

Reply:
I chose not to set a default on purpose, because this is a potentially very destructive command and I wanted the user to specify the upload source _explicitly_.